src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -53,7 +53,7 @@ case class RawlsBillingProject(projectName: RawlsBillingProjectName, users: Set[
 object UserAuth {
 
   def toWorkspaceAccessGroupName(workspaceName: WorkspaceName, accessLevel: WorkspaceAccessLevel) =
-    s"rawls ${workspaceName.namespace}/${workspaceName.name} ${accessLevel}"
+    s"${workspaceName.namespace}/${workspaceName.name} ${accessLevel}".take(64) // group names have a 64 char limit
```

Comment:
This is just the workspace name. It is for readability, we don't need to worry about collisions. The access level is available in the group email address so is probably the least important information in the name.